docs/services/nextcloud.md

```diff
@@ -273,7 +273,7 @@ You can also create new documents via the "plus" button.
 
 It is possible to setup preview generation, using this playbook.
 
-First modify your `vars.yml` file by adding at least the following line (other options are also present, check the corresponding `defaults/main.yml` file):
+First modify your `vars.yml` file by adding at least the following line (other options are also present, see below):
```

Comment:
I think the previous message (mentioning defaults/main.yml) was more useful as there are many other variables (not documented below) that users may find useful

Reply:
Good point, I reverted back that change. What do you think of its current form?